### PR TITLE
Informed consent

### PR DESCRIPTION
**Pull request by lorenzoPrimi**

### CODE REVIEW

**Comment by bassosimone** [comment]
@lorenzoPrimi could you apply the following changes to this branch:

1. make sure it points to the latest stable version of MK for Android `v0.3.2-1`

2. solve the conflicts that we have with master by updating to the most recent master, merging master into this branch, solving the conflicts, then pushing the merge commit? [as a suggestion for dealing with the merge, I'd investigate whether we really need to keep .iml files around; in another android app I am working on those are not committed into the repo and we have no issues whatsoever, so perhaps a sensible way forward is to just get rid of that pesky ide-generated files.]

Apart from that, I have smoke tested what you did in this branch and I think it was working okay. I'll also be grateful if you could open a issue stating that we should make sure the informed consent is tailored to a mobile app (similar to the issue I opened for netprobe-ios). After the conflicts has been resolved I'll continue the review of this branch looking more into the code details.

Thanks!

File: .idea/encodings.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="Encoding">
-    <file url="PROJECT" charset="UTF-8" />
-  </component>
-</project>
```
Comment (bassosimone) [inline .idea/encodings.xml]: Perhaps we can additionally add these removed files to .gitignore?

File: .idea/vcs.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```
Comment (bassosimone) [inline .idea/vcs.xml]: Also this file could be added to gitignore so we stop committing and tracking it?

File: app/src/main/java/org/openobservatory/netprobe/fragment/IConsentPage3Fragment.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        View v = inflater.inflate(R.layout.fragment_ic_page_3, container, false);
+        //WebView webview = (WebView)v.findViewById(R.id.wv);
+        //webview.getSettings().setJavaScriptEnabled(true);
+        //webview.loadDataWithBaseURL("", getString(R.string.quiz_text), "text/html", "UTF-8", "");
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/netprobe/fragment/IConsentPage3Fragment.java]: I'd be really happy if we can avoid merging commented out code :-)

**Comment by lorenzoPrimi** [comment]
@bassosimone I don't know how to manually merge, can you give me more instructions?

**Comment by bassosimone** [comment]
@lorenzoPrimi note that some of your commits are bound to an unrecognized email address (that however seems to be bound to another picture of you), perhaps because you local git configuration changed? (I suggest to fix either by changing you local git configuration or by telling github that also the other email address is owned by you).

**Comment by bassosimone** [comment]
@lorenzoPrimi yesterday I released `0.3.3-1`, would you mind point this branch to such latest version and do some smoke testing so that we can merge with master code using the latest version?

**Comment by lorenzoPrimi** [comment]
@bassosimone I added the other email to my git profile.
Better not merge this branch right now as I'm still working on this implementing the new tests.
Everything will be ready before the weekend

**Comment by bassosimone** [comment]
@lorenzoPrimi Ack! I've added the `on hold` label to indicate that the work is in progress but the branch is not to be considered ready for review yet (there is also a label for this). Thanks!

**Comment by lorenzoPrimi** [comment]
@bassosimone informed consent is finished.
For some reason test results is not shown, I'm debugging it and test log is always empty so I believe the tests are not ran.

**Pull request merged**